ui/js/controllers/bugfiler.js

```diff
@@ -0,0 +1,224 @@
+"use strict";
+
+treeherder.controller('BugFilerCtrl', [
+    '$scope', '$rootScope', '$uibModalInstance', '$http', 'summary', 'thBugzillaProductObject',
+    'thPinboard', 'thEvents', 'fullLog', 'parsedLog', 'reftest', 'selectedJob', 'allFailures',
+    'thNotify', 'ThLog',
+    function BugFilerCtrl(
+        $scope, $rootScope, $uibModalInstance, $http, summary, thBugzillaProductObject,
+        thPinboard, thEvents, fullLog, parsedLog, reftest, selectedJob, allFailures,
+        thNotify, ThLog) {
+
+        var $log = new ThLog("BugFilerCtrl");
+
+        /**
+         *  'enter' from the product search input should initiate the search
+         */
+        $scope.productSearchEnter = function(ev) {
+            if (ev.keyCode === 13) {
+                $scope.findProduct();
+            }
+        };
+
+        /*
+         **
+         */
+        $scope.isReftest = function() {
+            return reftest !== "";
+        };
+
+        /**
+         *  Pre-fill the form with information/metadata from the failure
+         */
+        $scope.initiate = function() {
+            $uibModalInstance.parsedSummary = $uibModalInstance.parseSummary(summary);
+
+            $scope.modalSummary = "Intermittent " + $uibModalInstance.parsedSummary[0].join(" | ");
+
+            $("#modalParsedLog").next().attr("href", parsedLog);
```

Comment:
Instead of doing this JQuery work here in the ``initiate`` function, you would do better to just set some scope variables outside this function in the main area like this:

    $scope.parsedLog = parsedLog;
    $scope.fullLog = fullLog;
    if ($scope.isReftest()) {
        $scope.reftest = reftest;
    }

Then in the HTML, for your ``<a`` elements, you just do  ``href="{{ fullLog }}"`` and it picks up the value from the scope.